android/common/src/main/java/com/revenuecat/purchases/common/mappers.kt

```diff
@@ -17,19 +17,28 @@ fun EntitlementInfo.map(): Map<String, Any?> =
         "isActive" to this.isActive,
         "willRenew" to this.willRenew,
         "periodType" to this.periodType.name,
+        "latestPurchaseDateMillis" to this.latestPurchaseDate.time / 1000.0,
```

Comment:
maybe a Date extension would make cleaner by killing duplication as in `toMillis()`

Reply:
yeah let's do it